pysize.py

```diff
@@ -0,0 +1,31 @@
+import sys
```

Comment:
Use this as a dependency

Reply:
Sorry, @webknjaz tell me more about this, please.

Reply:
pip install

Reply:
@webknjaz I don't find it on https://pypi.org/

Reply:
You could use:
1) git url
2) git submodule (a opposed to copy-paste)

Reply:
https://pip.pypa.io/en/stable/reference/pip_install/#git

Reply:
@webknjaz Thanks. I fixed it.